### PR TITLE
Add configurations to setup how controller will create a distribution

### PR DESCRIPTION
**Pull request by caiofralmeida**
Receive configurations from var env and setup as valid config struct.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,2 +1,12 @@
 LOG_LEVEL="debug"
 DEV_MODE="true"
+CF_PRICE_CLASS="PRICE_CLASS_ALL"
+CF_AWS_WAF="arn:aws:wafv2:us-east-1:123456789012:global/webacl/ExampleWebACL/473e64fd-f30b-4765-81a0-62ad96dd167a"
+CF_CUSTOM_SSL_CERT="arn:aws:acm:us-east-1:450112707314:certificate/589a0d1e-70bd-4244-bc34-5eba5ad13fc7"
+CF_SECURITY_POLICY="TLSv1.2_2021"
+CF_ENABLE_LOGGING="true"
+CF_S3_BUCKET_LOG="gympass-staging-cloudfront-access-log.s3.amazonaws.com"
```
Comment (LCaparelli) [inline .env.example:8]: Are these actual valid ARNs and bucket address from our accounts? We probably shouldn't keep these in version control

**Review by LCaparelli — COMMENTED** [review]
Awesome, thanks! Left a concern for sensitive data, can you please take a look?

File: .env.example (hunk 1 above)
Comment (caiofralmeida) [reply .env.example:8]: It makes sense. I'm going to change it, tkss

File: internal/config/config.go [hunk 2]
```diff
@@ -40,6 +64,30 @@ type OperatorCfg struct {
 	LogLevel string
 	// DevMode when set to "true" logs in unstructured text instead of JSON.
 	DevMode bool
+	// DefaultOriginDomain represents a valid domain to define in default origin.
+	DefaultOriginDomain string
+	// CloudFrontPriceClass determines how many edge locations CloudFront will use for your distribution.
+	// ref: https://docs.aws.amazon.com/cdk/api/latest/typescript/api/aws-cloudfront/priceclass.html
+	CloudFrontPriceClass string
```
Comment (LCaparelli) [inline internal/config/config.go:71]: Looks like you got the enum values from the TypeScript SDK hehe

For golang this should be `PriceClass_All`. Try Ctrl+F "PriceClassPriceClassAll" [here](https://docs.aws.amazon.com/sdk-for-go/api/service/cloudfront/)

Needs changing in the viper default and .env.example

**Pull request merged**